client/src/components/Login.js

```diff
@@ -40,57 +50,59 @@ export class Login extends Component {
       <Grid centered columns={2}>
         <Grid.Column>
           <Header as="h2" textAlign="center">
-            Login
+            Log-in to your account
           </Header>
-          <Segment>
+
+          <Segment secondary>
             <Form size="large" onSubmit={this.handleSubmit}>
               <Form.Field inline>
-                <Grid>
-                  <Grid.Column textAlign="center">
-                    <label>Email</label>
-                    <Input
-                      placeholder="nnn@nn.com"
-                      onChange={this.handleChange}
-                      name="email"
-                      value={email}
-                      type="email"
-                    />
-                  </Grid.Column>
-                </Grid>
+                <Input
+                  fluid="true"
+                  icon="user"
+                  iconPosition="left"
+                  placeholder="E-mail Address"
+                  onChange={this.handleChange}
+                  name="email"
+                  value={email}
+                  type="email"
+                />
               </Form.Field>
 
               <Form.Field inline>
-                <Grid>
-                  <Grid.Column textAlign="center">
-                    <label>Password</label>
-                    <Input
-                      placeholder="******"
-                      onChange={this.handleChange}
-                      name="password"
-                      value={password}
-                      type="password"
-                    />
-                  </Grid.Column>
-                </Grid>
+                <Input
+                  fluid="true"
+                  icon="lock"
+                  iconPosition="left"
+                  placeholder="Password"
+                  onChange={this.handleChange}
+                  name="password"
+                  value={password}
+                  type="password"
+                />
               </Form.Field>
 
-              <Grid>
-                <Grid.Column textAlign="center">
-                  {loginErr ? (
-                    <p>Something went wrong! Please check email or password!</p>
-                  ) : null}
-                  <Button type="submit" content="Submit" primary />
-                  <Button
-                    content="Cancel"
-                    secondary
-                    onClick={() => {
-                      this.props.history.push(`/`);
-                    }}
-                  />
-                </Grid.Column>
-              </Grid>
+              {loginErr ? (
+                <p>Something went wrong! Please check email or password!</p>
+              ) : null}
+              <Button
+                size="massive"
+                fluid="true"
+                type="submit"
+                content="Login"
+                primary
+              />
             </Form>
           </Segment>
+          <Segment size="huge" secondary textAlign="center">
```

Comment:
The text is a bit too big, can you use the normal size as we use in the rest of the website?